calico_containers/adapter/datastore.py

```diff
@@ -444,36 +463,112 @@ def get_bgp_peers(self, version):
         Get the configured BGP Peers
 
         :param version: "v4" for IPv4, "v6" for IPv6
-        :return: List of netaddr.IPAddress IP addresses.
+        :return: List of dictionaries:
```

Comment:
I agree - and I have this on my list of to-dos.  Any objection to me fixing this up later if I don't have time tonight -- that sounds like a bigger change that will need re-review.

If the only comments are docs strings and comments it'd be nice to get these changes in to tonights release.